### DISCUSSION TITLE
action.editWithVim filename

### DISCUSSION THREAD

**Discussion by alikve** [Jelmerro/Vieb#546] [Ideas]
I think it would be useful to call vim with some options depending on the site. For example, on a webmail it could be `vim -c "set filetype=mail textwidth=72 spell"`, on a forum `vim -c "set filetype=bbcode spell"` and so on.

Is it possible to do this?

I can imagine an external approach, for example, to have a script that first read the title of the active window (say, `xdotool getactivewindow getwindowname`) and then call vim with needed options.

But I think it would be better to be able to do this using Vieb, for example, Vieb could add the domain to the filename, something like `<unique random string>.example.org` (this is how it works in the ["Browser's external editor" Chrome extension](https://chrome.google.com/webstore/detail/browsers-external-editor/moakhilhbeednkjahjmomncgigcoemoi))

**Comment by Jelmerro** [comment]
This is a nice suggestion, I'll convert it to an issue.

**Comment by Jelmerro** [comment]
Please see https://github.com/Jelmerro/Vieb/issues/547

**Comment by Jelmerro** [comment]
This has been added in 12.5.0, please see the issue for details.
